enterprise/app/root/root.tsx

```diff
@@ -137,33 +142,37 @@ export default class EnterpriseRootComponent extends React.Component {
           </MenuComponent>
         )}
         <div className="page">
-          {sidebar && (
+          {sidebar && !code && (
             <SidebarComponent
               path={this.state.path}
               hash={this.state.hash}
               user={this.state.user}
               search={this.state.search}></SidebarComponent>
           )}
-          <div className="main">
+          <div className="root-main">
             {!this.state.loading && (
               <div className={`content ${login ? "content-flex" : ""}`}>
                 {invocationId && (
-                  <InvocationComponent
-                    user={this.state.user}
-                    invocationId={invocationId}
-                    key={invocationId}
-                    hash={this.state.hash}
-                    search={this.state.search}
-                    denseMode={this.state.denseMode}
-                  />
+                  <Suspense fallback={<div className="loading" />}>
```

Comment:
nit: add a TODO to make the rest of these `Suspense`ful as well?

Reply:
Done